lib/JITServer/JITServer.cpp

```diff
@@ -131,7 +131,13 @@ ServerInitializeThreadContext(
 {
     AUTO_NESTED_HANDLED_EXCEPTION_TYPE(static_cast<ExceptionType>(ExceptionType_OutOfMemory | ExceptionType_StackOverflow));
 
-    ServerThreadContext * contextInfo = HeapNew(ServerThreadContext, threadContextData);
+    ServerThreadContext * contextInfo = HeapNewNoThrow(ServerThreadContext, threadContextData);
```

Comment:
i'm pretty sure rpc handles the OOM and returns E_OUTOFMEMORY. but if we don't want to use that, maybe we should remove the AUTO_NESTED_HANDLED_EXCEPTION_TYPE?